### PR TITLE
increase and decrease by click working

### PR DESCRIPTION
**Pull request by controllhu**
Pls check thx bye

### CODE REVIEW

File: 02simplegolden/src/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+class App extends Component {    
+  constructor() {
```
Comment (gyorkibence) [inline 02simplegolden/src/index.js]: constructor is deprecated in new versions of react

File: 02simplegolden/src/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    super();
+    this.state = {
+      clicks: 0
+    }
+  }
+  
+  increase() {
```
Comment (gyorkibence) [inline 02simplegolden/src/index.js]: if the event handler method's are arrow functions, then you don't have to bind it

File: 02simplegolden/src/index.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  decrease() {
+    this.setState((prevState) => ({
+         clicks: prevState.clicks - 1
+      }));
+  }
+ 
+  render() {
+    return (
+      <div>
+        <button onClick={this.increase.bind(this)}>Buy one</button>
```
Comment (gyorkibence) [inline 02simplegolden/src/index.js]: this should be a Component with a button

File: 02simplegolden/src/index.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        <p>{this.state.clicks}</p>
```
Comment (gyorkibence) [inline 02simplegolden/src/index.js]: this should be a Display Component with a p tag

File: 02simplegolden/src/index.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        <button onClick={this.decrease.bind(this)}>Eat one</button>
+      </div>
+    );
+  }
+}
+
+ReactDOM.render(
```
Comment (gyorkibence) [inline 02simplegolden/src/index.js]: this should be in another file

File: 02simplegolden/src/index.js (hunk 1 above)
Comment (controllhu) [reply 02simplegolden/src/index.js]: constructor removed

File: 02simplegolden/src/index.js (hunk 2 above)
Comment (controllhu) [reply 02simplegolden/src/index.js]: bindings removed

File: 02simplegolden/src/index.js (hunk 3 above)
Comment (controllhu) [reply 02simplegolden/src/index.js]: button is now a component

File: 02simplegolden/src/index.js (hunk 4 above)
Comment (controllhu) [reply 02simplegolden/src/index.js]: display is now a component

File: 02simplegolden/src/index.js (hunk 5 above)
Comment (controllhu) [reply 02simplegolden/src/index.js]: separated to another file

**Pull request closed without merging**